deploy-renku/README.md

```diff
@@ -41,6 +41,8 @@ deploy-pr:
       renku_notebooks: "${{ needs.check-deploy.outputs.renku-notebooks }}"
       renku_ui: "${{ needs.check-deploy.outputs.renku-ui }}"
       renku_data_services: "${{ needs.check-deploy.outputs.renku-data-services }}"
+      amalthea: "${{ needs.check-deploy.outputs.amalthea }}"
+      amalthea-sessions: "${{ needs.check-deploy.outputs.amalthea-sessions }}"
```

Comment:
I think this needs to be `amalthea_sessions` as a key (with an underscore). This is easy to miss and has bit me in the past when adding data services